src/dmd/cond.d

```diff
@@ -149,13 +149,13 @@ extern (C++) final class StaticForeach : RootObject
         el = el.ctfeInterpret();
         if (el.op == TOK.int64)
         {
-            dinteger_t length = el.toInteger();
-            auto es = new Expressions();
-            foreach (i; 0 .. length)
+            const length = cast(size_t) el.toInteger();
```

Comment:
Doesn't work with cross compiler

Reply:
what is the name of the target size_t type ?

Reply:
`d_targ_size_t` or something like that.

Reply:
I can assert on the length before casting. Look at the context, such an amount of element would create an OOM anyway.

Reply:
Finally removed this change instead of speculating on what would work or not.